package.json

```diff
@@ -761,6 +761,11 @@
           "default": null,
           "description": "An array of strings that enable experimental features in the PowerShell extension."
         },
+        "powershell.developer.waitForSessionFileNumOfTries": {
```

Comment:
As a setting I think it would make more sense to expose a timeout and then in the implementation just retry until the total exceeds the timeout

Reply:
I like that.  Perhaps `powershell.developer.sessionStartupTimeout`?